src/main/java/org/tron/common/runtime/vm/program/Program.java

```diff
@@ -89,6 +89,8 @@
   //Max size for stack checks
   private static final int MAX_STACKSIZE = 1024;
 
+  public static BlockCapsule blockCap = null;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make blockCap a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1398&issues=AWWA_HwielKyZq-pP3d9&open=AWWA_HwielKyZq-pP3d9)

[ISSUE_KEY]: AWWA_HwielKyZq-pP3d9